cf-execd/cf-execd.c

```diff
@@ -561,7 +561,7 @@ static int SetupRunagentSocket()
         ret = snprintf(sock_info.sun_path, sizeof(sock_info.sun_path) - 1,
                        "%s/"CF_EXECD_RUNAGENT_SOCKET_NAME, RUNAGENT_SOCKET_DIR);
     }
-    if (ret <= (sizeof(sock_info.sun_path) - 1))
+    if ((size_t) ret <= (sizeof(sock_info.sun_path) - 1))
```

Comment:
Can `ret < 0`? Could add an assert or comment about this.

Reply:
Good idea !

Reply:
> Can `ret < 0`?

Not really. But `assert()` is OK, I guess.